tests/fixtures/ring_intercom_settings.json

```diff
@@ -0,0 +1,439 @@
+    "dynamic_network_switching_eligible": false,
+    "missing_key_delivery_address": false
+  },
+  "light_snooze_settings": {
+    "duration": 0
+  },
+  "cv_settings": {
+    "detection_types": {
+      "human": {
+        "enabled": false,
+        "mode": "none",
+        "notification": false
+      },
+      "loitering": {
+        "enabled": false,
+        "mode": "none",
+        "notification": false
+      },
+      "motion": {
+        "enabled": true,
+        "mode": "edge",
+        "notification": true
+      },
+      "moving_vehicle": {
+        "enabled": false,
+        "mode": "none",
+        "notification": false
+      },
+      "nearby_pom": {
+        "enabled": false,
+        "mode": "none",
+        "notification": false
+      },
+      "other_motion": {
+        "enabled": false,
+        "mode": "none",
+        "notification": false
+      },
+      "package_delivery": {
+        "enabled": false,
+        "mode": "none",
+        "notification": false
+      },
+      "package_pickup": {
+        "enabled": false,
+        "mode": "none",
+        "notification": false
+      }
+    },
+    "threshold": {
+      "loitering": 10,
+      "package_delivery": 2
+    }
+  },
+  "concierge_settings": {
+    "mode": "disabled",
+    "alexa_settings": {
+      "delay_ms": 10000
+    },
+    "autoreply_settings": {
+      "delay_ms": 10000
+    }
+  },
+  "schedule_settings": {},
+  "intercom_settings": {
+    "predecessor": "{\"make\":\"Comelit\",\"model\":\"2738W\",\"wires\":2}",
+    "config": "{\"intercom_type\": 2, \"number_of_wires\": 2, \"autounlock_enabled\": false, \"speaker_gain\": [-49, -35, -25, -21, -16, -9, -6, -3, 0, 3, 6, 9], \"digital\": {\"audio_amp\": 0, \"chg_en\": false, \"fast_chg\": false, \"bypass\": false, \"idle_lvl\": 32, \"ext_audio\": false, \"ext_audio_term\": 0, \"off_hk_tm\": 0, \"unlk_ka\": false, \"unlock\": {\"cap_tm\": 1000, \"rpl_tm\": 1500, \"gain\": 1000, \"cmp_thr\": 4000, \"lvl\": 25000, \"thr\": 30, \"thr2\": 0, \"offln\": false, \"ac\": true, \"bias\": \"h\", \"tx_pin\": \"TXD2\", \"ack\": 0, \"prot\": \"Comelit_SB2\", \"ask\": {\"f\": 25000, \"b\": 333}}, \"ring\": {\"cap_tm\": 40, \"rpl_tm\": 200, \"gain\": 2000, \"cmp_thr\": 4500, \"lvl\": 28000, \"thr\": 30, \"thr2\": 0, \"offln\": false, \"ac\": true, \"bias\": \"m\", \"tx_pin\": \"TXD2\", \"ack\": 0, \"prot\": \"Comelit_SB2\", \"ask\": {\"f\": 25000, \"b\": 333}}, \"hook_off\": {\"cap_tm\": 1000, \"rpl_tm\": 1500, \"gain\": 1000, \"cmp_thr\": 4000, \"lvl\": 25000, \"thr\": 30, \"thr2\": 0, \"offln\": false, \"ac\": true, \"bias\": \"h\", \"tx_pin\": \"TXD2\", \"ack\": 0, \"prot\": \"Comelit_SB2\", \"ask\": {\"f\": 25000, \"b\": 333}}, \"hook_on\": {\"cap_tm\": 1000, \"rpl_tm\": 1500, \"gain\": 1000, \"cmp_thr\": 4000, \"lvl\": 25000, \"thr\": 30, \"thr2\": 0, \"offln\": false, \"ac\": true, \"bias\": \"h\", \"tx_pin\": \"TXD2\", \"ack\": 0, \"prot\": \"Comelit_SB2\", \"ask\": {\"f\": 25000, \"b\": 333}}}}",
```

Comment:
Is this actually how the raw data is returned from the ring api query?  With the config escaped like this?

Reply:
Exactly, that's the raw data of my Intercom. I got it using the `raw-query` of the CLI.